std_msgs/Header header
string device_name
string status_messgae

diagnostic_msgs/KeyValue[] values